internal/pkg/keeper/api/caller.go

```diff
@@ -0,0 +1,34 @@
+//
+// Copyright (C) 2024 IOTech Ltd
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package api
+
+import (
+	"errors"
+
+	"github.com/edgexfoundry/go-mod-configuration/v3/internal/pkg/keeper/utils/http"
+)
+
+type Caller struct {
```

Comment:
Caller is too generic.
```suggestion
type KeeperApi struct {
```

Reply:
File removed, using the http clients from core-contract.